seedlet/assets/sass/structure/_responsive-logic.scss

```diff
@@ -86,14 +86,12 @@ $breakpoint_xxl: 1024px;
 	--responsive--alignwide-width: calc(100vw - var(--responsive--spacing-horizontal));
 	--responsive--alignfull-width: 100%;
 	--responsive--alignwide-width-multiplier: calc(16 * var(--global--spacing-horizontal));
-	--responsive--alignright-margin: var(--global--spacing-horizontal);
-	--responsive--alignleft-margin: var(--global--spacing-horizontal);
+	--responsive--alignrightleft-width: var(--global--spacing-horizontal);
```

Comment:
@jffng I decided to merge them into one to make it simpler since they are always the same anyway.